### DISCUSSION TITLE
New component proposal: AutoComplete

### DISCUSSION THREAD

**Discussion by draccagni** [carbon-design-system/carbon-components-svelte#1242] [Ideas]
A component like [AutoComplete](https://www.primefaces.org/primeng/autocomplete) would be very useful.

**Comment by dybxin** [comment]
@draccagni  you can refer the third-party implementation: https://github.com/pstanoev/simple-svelte-autocomplete

**Comment by metonym** [comment]
`ComboBox` now supports auto-complete with typeahead: https://svelte.carbondesignsystem.com/components/ComboBox#autocomplete-with-typeahead
